default.nix

```diff
@@ -117,14 +118,22 @@ let
       filter = localLib.isPlutus;
       requiredOverlay = ./nix/overlays/haskell-overrides.nix;
     };
-
     localPackages = localLib.getPackages {
       inherit (self) haskellPackages; filter = localLib.isPlutus;
     };
 
+    # # FIXME: currently only an approximation of the existing infra
+     haskell-packages-new = pkgs.plutusHaskellPackages;
+
+    local-packages-new = localLib.getPackages {
```

Comment:
I think this is fine for now. However, I think we could refactor this to just be the `haskell.nix` package set and pull things out as we need them.

Certainly for CI we want to build all tests, benchmarks, docs, etc.